internal/utils/templates/globals.sql

```diff
@@ -55,7 +53,7 @@ ALTER ROLE authenticated SET statement_timeout TO '8s';
 -- User Config "authenticator"
 --
 
-ALTER ROLE authenticator SET session_preload_libraries TO 'supautils', 'safeupdate';
```

Comment:
Any reason to change this?

Reply:
Yes, `supautils` extension is not installed on docker image. So this errors at runtime (more noticeable in self hosted setup).

Reply:
Note that we need `supautils` in docker eventually, when we move to [extension custom scripts](https://github.com/supabase/supautils/pull/26).

Reply:
Hopefully we will have centralised migrations by then :)